src/mkdocs/docs/guides/ssm-quick-setup-patching/ssm-quick-setup-patching-configuration.md

```diff
@@ -0,0 +1,143 @@
+
+Beginning December 22, 2022, Patch Manager offers a new, recommended method to configure patching for your organization and AWS accounts through the use of patch policies.
+
+A patch policy is a configuration you set up using Quick Setup. Patch policies provide more extensive and more centralized control over your patching operations than is available with previous methods of configuring patching. Patch policies can be used with all operating systems supported by Patch Manager, including supported versions of Linux, macOS, and Windows Server.
+
+Reference: https://docs.aws.amazon.com/systems-manager/latest/userguide/patch-manager-policies.html, https://docs.aws.amazon.com/systems-manager/latest/userguide/quick-setup-patch-manager.html
+
+## 1.2 Configuration Overview
+
+> The Quick Setup patch policy, described on this page, should only be configured AFTER a successful ASEA installation. 
+
+The Quick Setup patch policy is configured within the AWS Systems Manager console. The choices made in the configuration are used as inputs into two CloudFormation Stack Sets: Management ('MA'), and Target Accounts ('TA'). There are a few challenges that restrict the deployment and usage of the Quick Setup within the ASEA; thus, the purpose of creating this content. The challenges are:
+- The 'TA' Stack Set hardcodes a Retention on CloudWatch Log (CWL) Groups. CWL Groups are protected by the ASEA SCPs, so this StackSet deployment fails in its default configuration.
+- The patch policy configuration, from the Quick Setup, is saved in a S3 bucket, and is used by EC2 SSM Patching. Therefore, the EC2 IAM Instance Profile Roles must have S3 GetObject permissions on this specific S3 bucket.
+- That S3 bucket has a resource policy that denies all GetObject requests if the Role does not have a specific Tag and Tag value. Today, the ASEA config lacks the ability to apply tags to IAM Roles. As an example, the default ASEA configuration deploys 'EC2-Default-SSM-AD-Role' that would need Tags and S3:GetObject permissions.
+
+The solution, described in detail below, includes the following:
+- Manual modification of the 'TA' StackSet template to remove the hardcoded CWL retenion.
+- The creation of a policy, which contains a specific S3:GetObject, that is configured on IAM Roles.
+- The creation of a custom AWS Config Rule that inspects Tags on specific IAM Roles. If a target role does not have the specific Quick Setup patch policy Tag, the Role will be 'Non-Compliant'.
+- The creation of a SSM Document that can apply Tags to IAM Roles. This is used to auto-remediate a 'Non-Compliant' role detected by the previous custom AWS Config Rule.
+
+
+
+## 1.2 Deployment
+
+### 1.2.1 Quick Setup Patch Policy
+
+The following steps have been taken from the documentation [here](https://docs.aws.amazon.com/systems-manager/latest/userguide/quick-setup-patch-manager.html) with modifications.
+
+1. Open the AWS Systems Manager console at https://console.aws.amazon.com/systems-manager/.
```

Comment:
Add note that this need to be done in Management account

Reply:
Thanks for the feedback, I've updated the doc!